ActiveRecord.php

```diff
@@ -7,11 +7,16 @@
 if (!defined('PHP_ACTIVERECORD_AUTOLOAD_PREPEND'))
 	define('PHP_ACTIVERECORD_AUTOLOAD_PREPEND',true);
 
+if (PHP_VERSION_ID > 70100) {
+    require __DIR__.'/lib/DateTime71.php';
+} else {
+    require __DIR__.'/lib/DateTime.php';
+}

```

Comment:
Oh wow, that's the PERFECT way to handle this! I forgot that this library has its own "front loader" of sorts. 👏